src/kata5.reachDestination.js

```diff
@@ -1,3 +1,14 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  if (distance === 0 && speed === 0) {
+    return "I am here";
+  }
+  if (distance > 0 && speed === 0) {
+    return "The train has broken down";
+  }
+  const arrivalTime = distance / speed;
+  return `I will be there in ${Math.round(arrivalTime * 2) / 2} hours.`;
+};
 
 module.exports = reachDestination;
+
```

Comment:
Most linters will leave you a single blank line at the end of the file, should never be more than one either way